src/components/layout/Footer.tsx

```diff
@@ -23,7 +23,7 @@ const Footer = () => (
     expand="sm"
     sticky="bottom"
   >
-    <FooterContainer fluid>
+    <FooterContainer className="container" fluid>
```

Comment:
So `FooterContainer` is already a bootstrap `container`. We added `fluid`, which changes the underlying class the component gets from `container` to `container-fluid`. If you don't want it to be `container-fluid`, and want it to just be `container' instead, you can just remove the `fluid` attribute instead of adding the bootstrap class here. We want to avoid classes when we can and just use or extend components instead.